tests/test_actions/test_watch.py

```diff
@@ -194,7 +194,7 @@ def test_watch_no_folders(self):
         assert file_name2 not in self.action.doc_manager.get_names()
 
         #check in progress - get a list of remote files and ensure that the one that was supposed to be ignored isn't there
-        #print(str(self.action.api.list_documents(self.action.project_id)['entities']['properties']['title']))
+        #print(str(self.action.api.list_documents(self.action.project_id, self.action.community_id)['entities']['properties']['title']))
```

Comment:
Should this line be deleted since it's just a comment.

Reply:
@EPortela2013 We can, wasn't sure why it was commented out in the first place though